projects/plugins/debug-helper/changelog/add-waf-helper

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
Since this is a minor change I think we'll need to change the plugin version.

Reply:
Fixed! https://github.com/Automattic/jetpack/pull/29242/commits/d5da2efef31c2f3cbe8b99d037ec76356165a7bb